### ISSUE TITLE
Update or replace mock-django

### ISSUE THREAD

**Issue by Jawayria** [edx/upgrades#76]
We use the package [mock-django](https://github.com/dcramer/mock-django) in edx/ecommerce. This package isn't maintained now and it doesn't have support for Django3.2. Please follow the guidance in https://openedx.atlassian.net/wiki/spaces/AC/pages/3036972032/Handling+Outdated+Dependencies to resolve the problem this poses for the Open edX Django 3.2 upgrade.

**Comment by Jacatove** [comment]
Hi, I can help with this issue.

**Comment by jmbowman** [comment]
@Jacatove Thank you for reaching out.  I will assign this to you.  Please let me know once this is done.

**Comment by Jacatove** [comment]
Hi @jmbowman I have a question but first let me give you some context of what I have done.

1. I opened a [draft PR](https://github.com/edx/ecommerce/pull/3494). A solution based on the 5th option to [Handling Outdated Dependencies](https://openedx.atlassian.net/wiki/spaces/AC/pages/3036972032/Handling+Outdated+Dependencies). (There is more information explaining the reason I choose the 5th option in the PR description)
2. I have received an email from the maintainer of the dependency:
     _"no intention to maintain it on my side. Happen to give you maintainership though if thats interesting. Otherwise feel free to pull in the relevant code yall need. I can double check license to make sure theres no concerns, but I imagine its BSD/Apache-like."_

Now my questions:

Is 1. the better option for edx? 
    if yes, then where should I put the copied code from the dependency?

**Comment by Jacatove** [comment]
@diegomillan This is the ticket I need help with.

**Comment by UsamaSadiq** [comment]
Hi @Jacatove thanks for your efforts on the issue. Your PR has been merged. Since it is only a testing dependency so it'll be removed when merging the upgrade branch to master hence you won't need to create separate PR for that. 

I'll mark this issue as done. Thanks.
